extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcIdentityProvider.java

```diff
@@ -197,14 +197,15 @@ public Uni<SecurityIdentity> apply(TokenVerificationResult result, Throwable t)
 
     private Uni<TokenVerificationResult> verifyPrimaryTokenUni(Map<String, Object> requestData,
             TokenAuthenticationRequest request, TenantConfigContext resolvedContext, UserInfo userInfo) {
+        final Uni<TokenVerificationResult> tokenVerificationResult;
         if (isInternalIdToken(request)) {
             if (requestData.get(NEW_AUTHENTICATION) == Boolean.TRUE) {
                 // No need to verify it in this case as 'CodeAuthenticationMechanism' has just created it
-                return Uni.createFrom()
+                tokenVerificationResult = Uni.createFrom()
```

Comment:
I'd just leave this code as is and fail the build as proposed in another comment

Reply:
ok